### PR TITLE
Override properties with system properties

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Review by nineinchnick — APPROVED** [review]

File: provisio-maven-plugin/src/main/java/ca/vanzyl/maven/plugins/provisio/Provisio.java [hunk 1]
```diff
@@ -96,8 +96,7 @@ public List<Runtime> findDescriptorsForPackagingTypeInExtensionRealms(MavenProje
 
   public Runtime parseDescriptor(InputStream inputStream, MavenProject project) {
     RuntimeReader parser = new RuntimeReader(Actions.defaultActionDescriptors(), versionMap(project));
-    Map<String, String> variables = Maps.newHashMap();
-    variables.putAll((Map) getPropertiesWithSystemOverrides(project));
+    Map<String, String> variables = new HashMap<>(getPropertiesWithSystemOverrides(project));
```
Comment (nineinchnick) [inline provisio-maven-plugin/src/main/java/ca/vanzyl/maven/plugins/provisio/Provisio.java:99]: Missing import?

Comment (wendigo) [reply provisio-maven-plugin/src/main/java/ca/vanzyl/maven/plugins/provisio/Provisio.java:100]: Fixed

**Review by nineinchnick — APPROVED** [review]
Can you also improve the last commit's message? At least it should reference the latest commit (revert of revert)

**Comment by wendigo** [comment]
@nineinchnick done

**Pull request merged**